otx/mpa/modules/models/heads/custom_atss_head.py

```diff
@@ -36,6 +36,15 @@ def __init__(
         self.bg_loss_weight = bg_loss_weight
         self.use_qfl = use_qfl
 
+    def _init_layers(self):
+        """Initialize layers of the ATSS head.
+
+        Current mmdetection requires to call init_weights function to reflect init_cfg
+        """
+
+        super(CustomATSSHead, self)._init_layers()
+        self.init_weights()
+
```

Comment:
Is this a right way to call init_weghts()? (Sorry, I don't know, just question)
Is it different among other models & heads... or mmcv modules?

Reply:
You mean, we should check other structure and give general solution? I made this PR only for ATSS since, SSD and YOLOX does not give this kind of issue after mmcv framework change. I'll check whether other structure call init_weights. If they not call this func, then I'll find why those two structures work fine